apps/nextjs-website/src/components/Header.tsx

```diff
@@ -11,23 +11,34 @@ import {
   MegaHeaderData,
   HeaderSublink,
 } from '@/lib/fetch/header';
+import { Locale } from '@/lib/fetch/siteWideSEO';
+
+const BadgeTextValues = {
+  it: 'NOVITÀ',
+  en: 'NEW',
+  de: 'NEU',
+  fr: 'NOUVEAU',
+  sl: 'NOVO',
+};
 
 const makeSublink = (
   sublink: HeaderSublink,
-): {
-  label: string;
-  href: string;
-} => ({
+  locale: Locale,
+): { label: string; href: string; isNew: boolean; badge?: string } => ({
   label: sublink.label,
   href: sublink.page.data
     ? sublink.page.data.attributes.slug +
       (sublink.sectionID ? `#${sublink.sectionID}` : '')
     : (sublink.externalURL ?? ''),
+  isNew: sublink.isNew,
+  badge: sublink.isNew ? BadgeTextValues[locale] : '',
```

Comment:
```suggestion
  ...(sublink.isNew && { label: LinkLabelValues[locale] }),
```
If you always pass a value the check on badge's existence doesn't make sense.

We have to only pass badge if it has a value (which is when the sublink is new).

Also rename variables accordingly to be consistent with the change from NewBadge to LinkLabel.